CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+# Changelog
+
+<!-- Instructions
+
+This changelog follows the patterns described here: <https://keepachangelog.com/en/1.0.0/>.
+
+Subheadings to categorize changes are `added, changed, deprecated, removed, fixed, security`.
+
+-->
+
+## Unreleased
+
+### Changed
+
+- [#547](https://github.com/linebender/vello/pull/547) - `RenderContext::new()` no longer returns a `Result` by [@waywardmonkeys](https://github.com/waywardmonkeys)
```

Comment:
I think at least https://github.com/linebender/vello/pull/435, https://github.com/linebender/vello/pull/496, https://github.com/linebender/vello/pull/521, https://github.com/linebender/vello/pull/526, https://github.com/linebender/vello/pull/537, https://github.com/linebender/vello/pull/544, 
would be useful to have changelog entries. There are some other changes which probably should be included, but those are less important, and can be skipped

Reply:
Shouldn't it only make a changelog entry if it makes a difference to the user (e.g. rendering fix the user may be working around) or an API change?

I included everything except #496, because it seems that is just a checkpoint and doesn't make a tangible impact to users.

Reply:
Hmm. Not entirely sure what is right here. I don't think a performance gain fits into one of the changelog categories, but I'm happy to write it in to push this along.

Reply:
Done

Reply:
For some more context #496 resolved the "properly implementing strokes" blocker (#303) which was listed in the README. It is a quite significant change.